_data/versions.yml

```diff
@@ -1355,17 +1403,17 @@ master:
   note: ""
   components:
      felix:
-      version: 2.x-series
+      version: 2.6.0-8-gecbd512d
       url: ""
      calicoctl:
       version: v1.x-series
       url: "https://www.projectcalico.org/builds/calicoctl"
       download_url: "https://www.projectcalico.org/builds/calicoctl"
      calico/node:
-      version: v2.x-series
+      version: v2.6.x-series
```

Comment:
shouldn't this be a specific hash?

Reply:
This is the "master" section.  I think it's supposed to float at the "master" for this development branch.